src/test/java/seedu/address/logic/commands/FeedCommandTest.java

```diff
@@ -55,6 +58,8 @@
                     "https://blog.GNU.moe/anime/review/youjo_senki.html",
                     "I like this reviewer")
             );
+    private static final String MALFORMED_URL = "notavalidprotocol://malformed.url/invalid";
+    private static final String NOTAFEED_URL = "https://m4th.b0ss.net/temp/notafeed.notxml";
```

Comment:
We can link to the networktests gh-pages, see VALID_HTTPS_LINK in TypicalEntries

Reply:
I'll do that in the next PR (so every PR passes tests, perks of having my own web server 😃 ). But I will place a copy of these in gh-pages first.

Reply:
One of us will have to figure out asciidoctor before we can host raw files in GitHub pages, let's do it in the future.